rcgen/src/certificate.rs

```diff
@@ -843,6 +849,12 @@ impl CertificateParams {
 	}
 }
 
+impl AsRef<CertificateParams> for CertificateParams {
```

Comment:
Is this necessary? I would kind of expect that there's a blanket impl for this.

Reply:
Unfortunately, yes: https://doc.rust-lang.org/std/convert/trait.AsRef.html#reflexivity